tests/test_ingest_deck_cards.py

```
@@ -0,0 +1,98 @@
+@pytest.mark.parametrize("name,engine", _engines())
+def test_deck_cards_bulk_insert(name, engine):
+    session = _seed(engine)
+    try:
+        entry = session.get(TournamentEntry, "e1")
+        mainboard = [
+            {"Count": 4, "CardName": "Lightning Bolt"},
+            {"Count": 20, "CardName": "Island"},
+        ]
+        sideboard = [{"Count": 2, "CardName": "Lightning Bolt"}]
+
+        inserted, skipped, total = upsert_deck_cards_for_entry(
```

Ruff (0.15.20)

[warning] Unpacked variable `total` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)